src/gtk2.nim

```diff
@@ -6463,7 +6463,7 @@ proc in_drag*(a: PHandleBox): guint
 proc set_in_drag*(a: PHandleBox, `in_drag`: guint)
 proc shrink_on_detach*(a: PHandleBox): guint
 proc set_shrink_on_detach*(a: PHandleBox, `shrink_on_detach`: guint)
-proc snap_edge*(a: PHandleBox): gint
+#proc snap_edge*(a: PHandleBox): gint
```

Comment:
From the comment on line #13328:
set_snap_edge was in a when false. snap_edge put in when false as the actual return type should be GtkPositionType not gint